src/containers/AccessPointDetails/components/Firmware/index.js

```diff
@@ -35,6 +35,7 @@ const Firmware = ({ firmware, data, onUpdateEquipmentFirmware }) => {
   const handleUpdateFirmware = () => {
     onUpdateEquipmentFirmware(data.id, version.id);
     setRebootModal(false);
+    onFormUpdate('save');
```

Comment:
this is not what i wanted at all, you combined the wrong two props, combine onUpdateEquipmentFirmware and handleOnFormSave. If you don't understand what i'm saying to you ask more specific questions;  onUpdateEquipmentFirmware and handleOnFormSave are both the same workflow they are both called on save form, therefore they should be the same function instead of calling 2 functions to do the same thing one can.